src/components/configure/customkey/CustomKey.tsx

```diff
@@ -384,7 +396,9 @@ export default class CustomKey extends React.Component<OwnProps, OwnState> {
               onChange={(e) => {
                 this.onChangeHexCode(e.target.value);
               }}
+              onFocus={(evt) => evt.target.select()}
```

Comment:
@adamrocker Should the `select()` be called after 300ms here as well? Or, is it unnecessary because this is not an Autocomplete component?

Reply:
@yoichiro Yes. It works fine. Only the Autocomplete component does not work select() method on autofocus.

Reply:
@adamrocker I see. Thanks.